spec/integration/_dev/deploy/agent/custom-agent.spec.yml

```
@@ -13,12 +13,47 @@ spec:
         anyOf:
           - required:
             - hostname
+    volume:
+      type:
+        - object
+        - "null"
+      properties:
+        name:
+          type: string
+        driver:
+          type: string
+        driver_opts:
+          type: object
+          patternProperties:
+            "^.+$": 
+              type:
+                - string
+                - number
+        external:
+          type:
+            - boolean
+            - object
+          properties:
+            name:
+              deprecated: true
+              type: string
+          additionalProperties: false,
```

CI is failing due to something related to additionalProperties, it could be the comma here.
```suggestion
          additionalProperties: false
```